### DISCUSSION TITLE
Responsive props

### DISCUSSION THREAD

**Discussion by TrevorRice** [vanilla-extract-css/vanilla-extract#112] [Q&A] [answered]
I know vanilla-extract is lower level, but has anyone come up with a solution for handling responsive props? I'm looking to support the following prop API:

```
<Button size="sm">

or

<Button size={["sm", "md", "lg"]}>

or

<Button size={["sm", null, "lg"]}>
```

where the indices in the array correspond to the theme's defined breakpoints.

**Comment by mattcompiles** [comment, marked as answer]
Check out Sprinkles https://github.com/seek-oss/vanilla-extract/tree/master/packages/sprinkles
